src/lucky_migrator/create_table_statement.cr

```diff
@@ -118,6 +119,38 @@ class LuckyMigrator::CreateTableStatement
     end
   end
 
+  def default_value(type : String.class, default : String)
+    " DEFAULT '#{default}'"
+  end
+
+  def default_value(type : Int64.class, default : Int32 | Int64)
+    " DEFAULT #{default}"
+  end
+
+  def default_value(type : Int32.class, default : Int32)
+    " DEFAULT #{default}"
+  end
+
+  def default_value(type : Bool.class, default : Bool)
+    " DEFAULT #{default}"
+  end
+
+  def default_value(type : Float.class, default : Float)
+    " DEFAULT #{default.to_f}"
+  end
+
+  def default_value(type : Time.class, default : Time)
+    return " DEFAULT '#{default.to_utc}'"
```

Comment:
You can remove the `return` now that it's just this value

Reply:
Ah I didn't notice that